src/unix/notbsd/android/b64.rs

```diff
@@ -10,3 +11,7 @@ s! {
 }
 
 pub const SYS_gettid: ::c_long = 178;
+
+extern {
+    pub fn timegm(tm: *const ::tm) -> time64_t;
```

Comment:
Fixed. Looks like I'm just too used to Haskell, where circular dependencies are not supported :D